File: app/src/main/java/com/example/esraarashad/httpurlconnectionexample/detailspackage/model/ApiResponse.java
package com.example.esraarashad.httpurlconnectionexample.detailspackage.model;

import com.example.esraarashad.httpurlconnectionexample.detailspackage.model.ProfileModel.Profiles;

import java.util.ArrayList;

public interface ApiResponse {
    void onProcessFinished(ArrayList<Profiles> profiles);
}

File: app/src/main/java/com/example/esraarashad/httpurlconnectionexample/detailspackage/model/ApiResponse.kt
package com.example.esraarashad.httpurlconnectionexample.detailspackage.model

import com.example.esraarashad.httpurlconnectionexample.detailspackage.model.ProfileModel.Profiles

import java.util.ArrayList

interface ApiResponse {
    fun onProcessFinished(profiles: ArrayList<Profiles>)
}
